org.osgi.test.cases.framework/src/org/osgi/test/cases/framework/junit/context/InstallBundleTest.java

```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
```

Comment:
What does this have to do with the CM TCK?

Reply:
Nothing.

I just had some errors and had to find out whether this ins an tck Design error or something that came from Framework...